README.md

```diff
@@ -26,7 +26,7 @@ Cachet is a beautiful and powerful open source status page system.
 ## Requirements
 
 - PHP 5.5.9+ or newer
-- Apache or Nginx server
+- Http server with PHP support (eg: Apache, Nginx, Caddy)
```

Comment:
We could just cover all grounds and say `something that does some Http stuff with PHP, innit`